aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -889,6 +937,10 @@ func testAccPreCheckAWSEc2SpotFleetRequest(t *testing.T) {
 
 func testAccAWSSpotFleetRequestConfigBase(rName string, rInt int) string {
 	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {
+  state = "available"
+}
```

Comment:
After the creation of this PR, we enabled support for Local Zone management in the provider and in the testing of doing so in our testing accounts discovered that this data source was returning `us-west-2-lax-1a` as the first result, so we've been failures with missing instance types and functionality. Going to swap this here for the configuration that excludes Local Zones before running the testing.

```suggestion
data "aws_availability_zones" "available" {
  state = "available"

  filter {
    name   = "opt-in-status"
    values = ["opt-in-not-required"]
  }
}
```

This extra little configuration is now also in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#acceptance-testing-guidelines) so you can find it there in the future if you need to. 👍